tasks/33_array-plus-array/array-plus-array.js

```diff
@@ -0,0 +1,7 @@
+export function arrayPlusArray(array1, array2) {
+  let shorterArray = array1.length <= array2.length ? array1 : array2;
+  let longerArray = array1.length > array2.length ? array1 : array2;
```

Comment:
Do we really need to calculate the same condition twice?

Reply:
Fixed